console/console.go

```diff
@@ -216,6 +217,12 @@ func (c *Console) init(preload []string) error {
 	return nil
 }
 
+func (c *Console) clearHistory() {
+	c.history = nil
+	c.prompter.ClearHistory()
```

Comment:
Please call `os.Remove(c.histPath)` here.

Reply:
done. my original reasoning was that history is being persisted at the end of the session but I agree this is safer.